Makefile

```diff
@@ -32,7 +32,7 @@ lint: lint/go lint/protobuf ## Execute linters.
 
 .PHONY: lint/go
 lint/go: ## Execute golangci-lint.
-	@go run github.com/golangci/golangci-lint/cmd/golangci-lint run
+	@go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.47.2 run
```

Comment:
https://github.com/golangci/golangci-lint/issues/2649
-> By not fixing a version here, ou lint was breaking with misleading issues

Reply:
Thanks for discovering that 🎊